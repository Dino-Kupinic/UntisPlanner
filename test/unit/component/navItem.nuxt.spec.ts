import {describe, test, it, expect, vi} from "vitest"
import {mountSuspended} from "@nuxt/test-utils/runtime"
import {NavItem} from "#components"

describe("NavItem", async () => {
  const component = await mountSuspended(NavItem)

  test("it can mount properly", async () => {
    expect(component.text()).toMatchSnapshot()
  })

  it("contains a UButton", async () => {
    expect(component.findComponent("UButton")).toBeTruthy()
  })
})
