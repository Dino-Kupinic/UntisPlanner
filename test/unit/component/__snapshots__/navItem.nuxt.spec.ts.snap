// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NavItem > it can mount properly 1`] = `""`;
